src/IngredientList.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import './IngredientList.scss';
+
+function IngredientList({ recipe }) {
+
+    const ingredientKeys = Object.keys(recipe);
+
+    let hash = {
```

Comment:
this works too. I was thinking more like 
```
[
  {
    ingedient: Vanilla,
    measure: 2tsp
  },
  {
     ingredient: cinnamon,
     measure: 1tbsp
  },
  etc...
]
```

Reply:
ah ok. that makes more sense.